packages/StepTracker/StepTracker.modules.scss

```diff
@@ -0,0 +1,83 @@
+@import '~@tds/core-colours/colours';
+@import '~@tds/core-responsive/responsive';
+
+.container {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+}
+
+.step {
+  width: 100%;
+  position: relative;
+  text-align: center;
+
+  &::before,
+  &::after {
+    position: absolute;
+    top: 17px;
+    width: calc(50% - 1.07rem);
+    content: ' ';
+    border-bottom: 0.1rem solid $color-shuttle-grey;
+  }
+
+  &::before {
+    left: 0;
+  }
+
+  &::after {
+    right: 0;
+  }
+
+  .icon {
+    display: inline-block;
+    border: 0.1rem solid $color-shuttle-grey;
+    border-radius: 50%;
+    font-size: 1.2rem;
+    width: 35px;
+    height: 35px;
+    margin-bottom: 1rem;
+  }
+
+  .label {
+    position: relative;
+    width: 100%;
+    display: none;
+    text-align: center;
+
+    @include mq($from: md) {
+      display: block;
```

Comment:
Perhaps we should extract some reusable styles/classes for showing and hiding content at breakpoints. We have the same concept in ExpandCollapse now too. See [PanelWrapper.modules.scss](https://github.com/telusdigital/tds-core/blob/master/packages/ExpandCollapse/PanelWrapper/PanelWrapper.modules.scss#L25-L39)

Reply:
Sounds like a good idea. Since it impacts that component as well, should that be part of this pull request? I feel like it could be its own thing.

Reply:
Up to you how you would want to manage that.

Reply:
This will be addressed in a later story: https://telusdigital.atlassian.net/browse/TDS-633